syntax = "proto3";

package serialization;

message FullName {
    string surname = 1;
    string name = 2;
    string patronymic = 3;
}

message Student {
    FullName fullName = 1;
    repeated int32 grades = 2;
    int32 averageGrade = 3;
}

message StudentsGroup {
    repeated Student students = 1;
}
